app/views/physical_server/show.html.haml

```diff
@@ -1,5 +1,12 @@
+- @angular_form = true
```

Comment:
Skipping `app/views/layouts/_x_edit_buttons.html.haml` seems to be the only real purpose.

And since this is not an Add/Edit form, the question of rendering buttons does not arise here.
This declaration `- @angular_form = true` can be removed from here.